tests/e2e/e2e.sh

```diff
@@ -90,10 +46,18 @@ do
 	echo "---" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
 done
 
-kubectl get pods --all-namespaces -o wide
+#kind cluster setup
+pushd "${ISTIO_DIR}"
+# shellcheck disable=SC1091
+source "./prow/lib.sh"
```

Comment:
not sure what this is or does.  Did you add prow/lib.sh to the repo?  If its already in repo - make sure it does what you expect it to. @howardjohn and I were speaking of the need to generalize this work so it can be reused around all satellite repos (cni and operator to be specific at first) - although that generalization work can follow on making it work for the specific case of operator - which is essential for the release.  If your struggling, I should be available for a conversation tonight on webex.

Reply:
It is part of istio repo. So we can resue all the functions there.

Reply:
disagree - operator should stand alone in its dependencies on building the repository.  If we need a common files prow.ib, we can do that, but it should be merged as a separate PR in common-files repo (see link above).

Reply:
see review for more information about why using a prow library is an undesirable approach.  As painful as it may be, please copy the parts you need into this e2e.sh script.  As followup work a refactor of the e2e for the various repos (istio/operator, istio/cni, istio/istio) can be done in https://github.com/istio/common-files.